CMakeLists.txt

```
@@ -2301,27 +2301,26 @@ endif()
 # FFmpeg 4.x support
 # FFmpeg is multimedia library that can be found http://ffmpeg.org/
 find_package(FFMPEG COMPONENTS libavcodec libavformat libavutil libswresample)
-cmake_dependent_option(FFMPEG "FFmpeg 4.x support" ON "FFMPEG_FOUND;FFMPEG_libavcodec_VERSION VERSION_GREATER_EQUAL 58" OFF)
+cmake_dependent_option(FFMPEG "FFmpeg 4.x support" ON "FFMPEG_FOUND" OFF)
```

Ubuntu build fails. I suggest:

```suggestion
option(FFMPEG "FFmpeg 4.x support" OFF)
```

---

No exception for outdated Ubuntu LTS versions. You have to disable FFmpeg explicitly if it is installed. All cmake_dependent_option() features should be consistent now.